tutorials/02_ Run_a_workflow.md

```diff
@@ -147,57 +147,57 @@ We now have our structure ready to go! Let's get back to running a workflow.
+Next, look at a more advanced calculation. Run the command `simmate workflows show-config energy_mit`. Here, you'll see that some INCAR settings rely on composition and that we have a list of error handlers to help ensure that the calculation finishes successfully.
 
-So we can now view the settings used for *all* compositions/structures, but now we want to give it a specific structure. We'll give it the POSCAR structure we just made. To do this, make sure our terminal has the same folder open as where our file is! For example, if your POSCAR is on your Desktop while your terminal is in your home directory, you can run `cd Desktop` to change your active folder to your Desktop. Then run the command `simmate workflows setup-only energy_mit POSCAR`. You'll see a new folder created named `MIT_Static_Energy_inputs`. When you open it, these are all of the files that Simmate made for VASP to use. This is useful when you're an advanced user that wants to alter these files before calling VASP manually -- this happens often when testing out new workflows or unique systems.
+Now, let's go one step further and feed a specific structure (the POSCAR we just made) into a specific workflow (energy_MIT). To do this, make sure our terminal has the same folder open as where our file is! For example, if your POSCAR is on your Desktop while your terminal is in your home directory, you can type `cd Desktop` to change your active folder to your Desktop. Then run the command `simmate workflows setup-only energy_mit POSCAR`. You'll see a new folder created named `MIT_Static_Energy_inputs`. When you open it, you'll see all the files that Simmate made for VASP to use. This is useful when you're an advanced user who wants to alter these files before running VASP manually -- this could happen when you want to test new workflows or unique systems.
 
-For absolute beginners, you don't need to need to immediately understand these files/settings, but they will be important for knowing the scientific limitations of your results and for running your own custom calculations. For whichever program you end up using the most, be sure to also go through their tutorials -- rather than always using Simmate to run their program for you. For example, if you use VASP the most, you can learn to use VASP independently with [their tutorials](https://www.vasp.at/wiki/index.php/Category:Tutorials). Until you reach that point, we'll have Simmate do it all for us!
+For absolute beginners, you don't immediately need to understand these files, but they will eventually be important for understanding the scientific limitations of your results or for running your own custom calculations. Whether you use [VASP](https://www.vasp.at/wiki/index.php/Category:Tutorials), abinit, or another program, be sure to go through their tutorials, rather than always depending on Simmate to run the program for you.  Until you reach that point, we'll have Simmate do it all for us!
```

Comment:
```suggestion
For absolute beginners, you don't immediately need to understand these files, but they will eventually be important for understanding the scientific limitations of your results or for running your own custom calculations. Whether you use [VASP](https://www.vasp.at/wiki/index.php/Category:Tutorials), [ABINIT](https://docs.abinit.org/tutorial/), or another program, be sure to go through their tutorials, rather than always depending on Simmate to run the program for you.  Until you reach that point, we'll have Simmate do it all for us!
```